tasks/section5.yml

```diff
@@ -439,7 +589,7 @@
       - rule_5.2.14
 
 - name: "SCORED | 5.2.15 | AUDIT | Ensure SSH access is limited"
-  command: /bin/true
+  command: grep "^AllowGroups" /etc/ssh/sshd_config
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*